test/async-hooks/test-fseventwrap.mjs

```diff
@@ -0,0 +1,42 @@
+import { isMainThread, skip, isIBMi } from '../common/index.mjs';
+
+import { strictEqual } from 'assert';
+import initHooks from './init-hooks.mjs';
+import tick from '../common/tick.js';
+import { checkInvocations } from './hook-checks.mjs';
+import { watch } from 'fs';
+import process from 'process';
+import { fileURLToPath } from 'url';
+
+const __filename = fileURLToPath(import.meta.url);
```

Comment:
nit: converting to a path is not necessary, you could use the file URL directly.
```suggestion
const __filename = new URL(import.meta.url);
```